src/test/java/uk/nhs/ctp/auditFinder/AWSAuditFinderTest.java

```diff
@@ -70,28 +64,22 @@
 
   @Test
   public void findAll_withNullClient_shouldFail() {
-    var authService = mock(TokenAuthenticationService.class);
-    when(authService.requireSupplierId()).thenReturn("test-supplier");
-    var auditFinder = new AWSAuditFinder(null, authService, mock(ObjectMapper.class));
+    var auditFinder = new AWSAuditFinder(null, authService, objectMapper);
 
     expectedException.expect(IllegalArgumentException.class);
```

Comment:
Will this ever be the case? Specifying a null esClient seems unexpected, and presumably Spring injection will fail if there isn't one in the context?

Reply:
This is true, the ES client itself makes sure that a URL is specified. I'll remove these tests.